spec/fixtures/grpc/hello.proto

```diff
@@ -17,5 +17,5 @@ message HelloRequest {
 }
 
 message HelloResponse {
-  required string reply = 1;
+  string reply = 1;
```

Comment:
Let's not modify existing fields. Also, please make sure to add a new, separate, _test case_ that will leverage this new feature.